### PR TITLE
cmdct-3387 autocomplete CPA-AD for 2024

### PR DESCRIPTION
**Pull request by britt-mo**
### Description
 autocomplete CPA-AD for 2024

### Related ticket(s)

CMDCT-3387

---
### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by ailZhou — CHANGES_REQUESTED** [review]
Everything looks good expect some text is missing:

Screenshot 2024-03-18 at 9 07 49 AM

There should be a performanceMeasureSubtext prop you can add this text to.

File: services/ui-src/src/measures/2024/CPAAD/index.tsx [hunk 1]
```diff
@@ -1,43 +1,16 @@
...
-          <Q.DataSource />
-          <Q.DefinitionOfPopulation />
-          <Q.PerformanceMeasure />
-        </>
-      )}
-      <CMQ.AdditionalNotes />
-    </>
+    <QMR.AutocompletedMeasureTemplate
+      year={year}
+      measureTitle={`CAP-AD - ${name}`}
```
Comment (ailZhou) [inline services/ui-src/src/measures/2024/CPAAD/index.tsx:11]: i think there's a misspelling here, should be `{`CPA-AD - ${name}`}`

Comment (britt-mo) [reply services/ui-src/src/measures/2024/CPAAD/index.tsx:11]: 🤦🏻‍♀️ omg

**Review by ailZhou — DISMISSED** [review]
lgtm! changes are updated and CPA-AD is autocompleted by default

**Review by benmartin-coforma — DISMISSED** [review]

**Review by benmartin-coforma — APPROVED** [review]

**Pull request closed without merging**